config/schema/google_analytics_counter.schema.yml

```diff
@@ -1,24 +1,54 @@
+# Schema for the configuration files of the google_analytics_counter module.
+
 google_analytics_counter.settings:
-  type: mapping
-  label: Settings
+  type: config_object
+  label: 'Google Analytics Counter settings'
   mapping:
     cron_interval:
       type: integer
+      label: 'Minimum time between Google Analytics data fetching'
     chunk_to_fetch:
       type: integer
+      label: 'Number of items to fetch from Google Analytics in one request'
     api_dayquota:
       type: integer
+      label: 'Number of items to fetch from Google Analytics in one request'
     cache_length:
       type: integer
+      label: 'The minimum time in hours to elapse between getting fresh data for the same query from Google Analytics'
     start_date:
       type: string
+      label: 'The start date for GA queries'
     overwrite_statistics:
       type: boolean
+      label: 'Override the counter of the core statistics module'
     profile_id:
       type: string
+      label: 'The Google Analytics profile to use'
     client_id:
       type: string
+      label: 'Client ID created for the app'
     client_secret:
       type: string
+      label: 'Client Secret created for the app'
     redirect_uri:
       type: string
+      label: 'Used to override the host for the callback uri'
+    total_hits:
+      type: integer
+      label: 'The total number of hits registered by Google Analytics under this profile'
+    total_nodes:
```

Comment:
This should not be configuration.
In general anything that cron or editors on a site can change should not be config but state or content.
So the number of nodes or the total hits are good candidates for state as they are calculated by cron depending on the external api or the amount of content on the site.
config is only settings that a developer wants to deploy. They are never to be changed on a production site other than by deploying the config.